core/src/main/scala/com/spingo/op_rabbit/ConnectionParams.scala

```diff
@@ -53,26 +55,24 @@ case class ConnectionParams(
     factory.setRequestedFrameMax(requestedFrameMax)
     factory.setRequestedHeartbeat(requestedHeartbeat)
     factory.setSaslConfig(saslConfig)
-    sharedExecutor foreach factory.setSharedExecutor
+    sharedExecutor.foreach(factory.setSharedExecutor)
```

Comment:
seems unrelated but okay :D

Reply:
IntelliJ IDEA Scala plugin improving the style check, and I'm also improving the code style following the recommendations :)